test/gtest/ucp/test_ucp_tag_match.cc

```diff
@@ -446,7 +450,46 @@ UCS_TEST_P(test_ucp_tag_match, sync_send_unexp) {
     request_release(my_send_req);
 }
 
-UCS_TEST_P(test_ucp_tag_match, sync_send_unexp_rndv, "RNDV_THRESH=1048576") {
+UCP_INSTANTIATE_TEST_CASE(test_ucp_tag_match)
+
+class test_ucp_tag_match_rndv : public test_ucp_tag_match {
+public:
+    enum {
+        RNDV_SCHEME_AUTO = 0,
+        RNDV_SCHEME_PUT_ZCOPY
+    };
+
+    static const std::string rndv_schemes[];
+
+    void init() {
+        ASSERT_LE(GetParam().variant, (int)RNDV_SCHEME_PUT_ZCOPY);
+        modify_config("RNDV_SCHEME", rndv_schemes[GetParam().variant]);
+
+        test_ucp_tag_match::init();
+    }
+
+    std::vector<ucp_test_param>
+    static enum_test_params(const ucp_params_t& ctx_params,
+                            const std::string& name,
+                            const std::string& test_case_name,
+                            const std::string& tls)
+    {
+        std::vector<ucp_test_param> result;
+        generate_test_params_variant(ctx_params, name,
+                                     test_case_name + "/rndv_" +
+                                     rndv_schemes[RNDV_SCHEME_AUTO],
+                                     tls, RNDV_SCHEME_AUTO, result);
+        generate_test_params_variant(ctx_params, name,
+                                     test_case_name + "/rndv_" +
+                                     rndv_schemes[RNDV_SCHEME_PUT_ZCOPY],
+                                     tls, RNDV_SCHEME_PUT_ZCOPY, result);
```

Comment:
maybe add GET as well (for those who supports it)?

Reply:
@brminich yes, it's good idea
I copied the same approach as in `test_ucp_tag_xfer.cc` - looks like they only use "auto" and "put_zcopy" to not test GET Zcopy twice.
Only TCP doesn't support GET Zcopy, will check a way on how to check the support for a specific scheme before ucp_init() done

Reply:
you can generate variants for all and then skip if not supported

Reply:
where do u check if certain RNDV scheme is supported?

Reply:
> where do u check if certain RNDV scheme is supported?

actually, there are no needs to check it, since UCP has a fallback on SW RNDV if certain RNDV scheme is not supported